src/libs/actions/IOU.js

```diff
@@ -190,16 +218,16 @@ function rejectTransaction({
     })
         .then((response) => {
             if (response.jsonCode !== 200) {
-                throw new Error(`${response.code} ${response.message}`);
+                Log.hmmm('Error rejecting transaction', {error: response.error});
```

Comment:
NAB: Should we be including the error codes in these log messages?

Reply:
Maybe.. I didn't think super hard about this one, but I feel like we should be able to see the request and what error was thrown along with this log line and piece together what happened. This log line felt overly cautious or possibly unnecessary but I wasn't sure.

I think in the future we could consider having a default log any time an action returns a non-200 jsonCode

Reply:
> I think in the future we could consider having a default log any time an action returns a non-200 jsonCode

It would certainly make debugging easier if we did.